### PR TITLE
Fix PostgresColumnType.interval typo

### LINKED ISSUE #193: Column type typo?
I'm not sure about this, since I'm not using that case, but this looked quite odd to me and I couldn't find any documentation about it, that it might be a typo:

`case .interval: return "INTEVERAL"` in [Line 518 in PostgresColumnType.swift](https://github.com/vapor/postgres-kit/blob/master/Sources/PostgresKit/PostgresColumnType.swift#L518).

### PR DESCRIPTION
**Pull request by code28**
Fixes typo in `PostgresColumnType.interval`'s string serialization (#194, fixes #193).

### CODE REVIEW

**Review by 0xTim — APPROVED** [review]
LGTM but I'll wait for another review from Tanner to merge

**Comment by tanner0101** [comment]
These changes are now available in [2.1.1](https://github.com/vapor/postgres-kit/releases/tag/2.1.1)

**Pull request merged**
